package.json

```diff
@@ -34,54 +47,30 @@
     "graphql": "^14.5.8",
     "graphql-tag": "^2.10.1",
     "husky": "^3.0.3",
+    "leaflet": "^1.5.1",
+    "lerna": "^3.20.2",
     "lint-staged": "^9.2.1",
     "prettier": "^1.18.2",
+    "prop-types": "^15.7.2",
     "react": "^16.10.2",
     "react-dom": "^16.10.2",
     "webpack": "^4.39.2"
   },
-  "main": "core/index.js",
   "scripts": {
-    "preversion": "yarn lint && yarn build",
-    "postversion": "yarn publish --access public --new-version $npm_package_version . && git push --tags && (git diff-index --quiet HEAD || git push) && echo \"Successfully released version $npm_package_version!\"",
-    "postpublish": "yarn deploy",
-    "lint": "yarn eslint --fix --ignore-path './.gitignore' --ext '.js,.json,.md' './'",
-    "lint-staged": "yarn eslint --fix --ignore-path './.gitignore' --ext '.js,.json,.md'",
+    "preversion": "yarn lint",
+    "postversion": "NODE_ENV=production yarn build",
```

Comment:
Not sure I follow the reasoning here:

`lerna publish` calls `lerna version` and each of these call life-cycles methods on root as well each package ([`version`](https://github.com/lerna/lerna/tree/master/commands/version#lifecycle-scripts) and [`publish`](https://github.com/lerna/lerna/tree/master/commands/publish#lifecycle-scripts)).

This means we can `lint`, `test` and `build` to make sure everything is correct in `preversion`. We can then do the copy either in `postversion` or `prepack` hooks. This way things are consistent: bump version if everything is OK rather than bump version and hope for the best.